docs/metrics.md

```diff
@@ -0,0 +1,51 @@
+## Scraping and storing metrics for the run
+
+In addition to supporting tracking the metrics by deploying pre-loaded grafana dashboards on the live cluster, Kraken also supports capturing metrics for the duration of the scenarios defined in the config and indexes then into Elasticsearch to be able to store and evaluate the state of the runs long term. The indexed metrics can be visualized with the help of Grafana. It uses [Kube-burner](https://github.com/cloud-bulldozer/kube-burner) under the hood. The metrics to capture need to be defined in a metrics profile which Kraken consumes to query prometheus ( installed by default in OpenShift ) with the start and end timestamp of the run. Each run has a unique identifier ( uuid ) and all the metrics/documents in Elasticsearch will be associated with it. The uuid is generated automatially if not set in the config. This feature can be enabled in the [config](https://github.com/cloud-bulldozer/kraken/blob/master/config/config.yaml) by setting the following:
```

Comment:
indexes the**m** into Elasticsearch (end of second line) 

Might just be me but the first part of this blob is a little hard to read, very wordy. Not sure how we can make it easier to read but still get all the information. Maybe split into 2 sentences?

Reply:
Sure thing.

Reply:
@paigerube14 I split the blob into two sentences like you suggested. PTAL.